CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ The changelog is available [on GitHub][2].
   Drop support of GHC-8.2.2.
 * [#252](https://github.com/kowainik/tomland/issues/252):
   Move to `hspec-*` family of libraries from `tasty-*`.
+* [#256](https://github.com/kowainik/tomland/issues/256):
+  Rename modules to simplify module structure.
+  __TODO__: write migration guide
```

Comment:
I think you can add that you renamed `ParseException` to `TomlParseError` in the CHANGELOG as well :slightly_smiling_face: